functions/vpn.bats

```diff
@@ -3,29 +3,34 @@
 load vpn.bash
 load helpers.bash
 
+setup_file() {
+  export BASEDIR="${BATS_TEST_DIRNAME}/.."
+}
+
 teardown_file() {
   systemctl kill wg-quick@wg0.service || true
 }
 
-@test "destructive-wireguard_install" {
+@test "development-wireguard_install" {
   echo -e "# ${COL_CYAN}$(timestamp) [openHABian] Wireguard VPN installation starting...${COL_DEF}" >&3
+  setup_file
   run install_wireguard install 3>&-
-  #if [ "$status" -ne 0 ]; then echo "$output" >&3; fi
-  echo "$output" >&3
+  if [ "$status" -ne 0 ]; then echo "$output" >&3; fi
   [ "$status" -eq 0 ]
   echo -e "# ${COL_GREEN}$(timestamp) [openHABian] Wireguard VPN installation successful.${COL_DEF}" >&3
 
   echo -e "# ${COL_CYAN}$(timestamp) [openHABian] Wireguard VPN setup starting...${COL_DEF}" >&3
   run setup_wireguard 3>&-
-  if [ "$status" -ne 0 ]; then echo "$output" >&3; fi
+  echo "$output" >&3
   [ "$status" -eq 0 ]
   echo -e "# ${COL_GREEN}$(timestamp) [openHABian] Wireguard VPN setup successful.${COL_DEF}" >&3
 
-  echo -e "# ${COL_CYAN}$(timestamp) [openHABian] Checking if Wireguard service is running...${COL_DEF}" >&3
-  run systemctl is-active wg-quick@wg0.service 3>&-
-  if [ "$status" -ne 0 ]; then echo "$output" >&3; fi
-  [ "$status" -eq 0 ]
-  echo -e "# ${COL_GREEN}$(timestamp) [openHABian] Wireguard service is running.${COL_DEF}" >&3
+  # that will not work inside Docker ...
```

Comment:
Why not?

Reply:
Loading kernel modules does not seem to work in Docker hence the service doesn't run.
BTW changing kernel parameters doesn't either (it's refused as "read-only").

Reply:
Hmm, nothing is ever perfect, oh well guess we will need to make do with what works.